enn_ppo/enn_ppo/test_training.py

```diff
@@ -70,17 +70,18 @@ def test_not_hotdog() -> None:
     assert meanrew >= 0.99
 
 
-def test_count() -> None:
+def test_masked_count() -> None:
```

Comment:
let's still keep the previous test with `Count` as well

Reply:
The reason this was removed, was because the environment itself was hardcoded to only allow two possible outcomes (So it could pass the test in reasonable time).

Making the number of possible outcomes configurable was not simple as it would change the action-space which is class-level meaning I would have to implement a class factory method like I did for Griddly, which seems a bit overcomplicated for a simple environment like this.

Instead I opted for testing using the masked count (with a very reduced mask) so that the test would run in reasonable time, but essentially be similar to the previous test as it forces only two options.